File: udemy/java/salario/src/salario/Salario.java
package salario;
import javax.swing.JOptionPane;
public class Salario {

	public static void main(String[] args) {
		float salario_bruto, salario_liquido,inss;
		String nome;
		nome = JOptionPane.showInputDialog("informe o seu nome: ");
		salario_bruto = Float.parseFloat(JOptionPane.showInputDialog("informe o seu salario bruto"));
		inss = (salario_bruto * 9)/100;
		salario_liquido = salario_bruto - inss;
		JOptionPane.showMessageDialog(null,"ola "+nome+
											" \nseu salario bruto eh: r$ " +salario_bruto+
											" \nseu inss eh: r$ "+inss+
											" \nseu salario liquido eh: r$ "+salario_liquido);
	}

}

File: udemy/pascal/projetos/Salario.pas
program salario;
var salario_bruto, salario_liqui, inss:real;
nome:string;
begin writeln ('informe o seu nome: ');
read (nome);
writeln ('informe o seu salario bruto: ');
read (salario_bruto);
inss:= (salario_bruto * 9) / 100;
salario_liqui:= salario_bruto - inss;
writeln ('OlC! ', nome);
writeln ('seu salario bruto eh: ', salario_bruto: 6: 2, ' seu salario_liqui eh: ', salario_liqui: 6: 2, ' o valor do INSS eh: ', inss: 6:2) end.
